validate-swupd.sh

```diff
@@ -50,6 +50,13 @@ for i in $(seq $1 10 $2); do
 		popd
 		exit 1
 	fi
+	echo "helloworld" >> local-packages
+	mixer bundle add helloworld
+	if [[ $? -ne 0 ]]; then
+		echo "failed to add mix bundles"
+		popd
```

Comment:
No point in changing directory via popd if you are going to exit.

Reply:
Done